letsencrypt/config.yaml

```diff
@@ -91,4 +91,6 @@ schema:
     inwx_username: str?
     inwx_password: str?
     inwx_shared_secret: str?
+  elliptic_curve: "list(secp256r1|secp384r1|secp521r1)?"
```

Comment:
This should not be an option. Addons are not supposed to present every possible option to user, especially not hyper technical ones such as these. Whenever possible, addon devs should preselect the most sensible option for users so they don't have to think about it. This is one of those cases, please default to the best option and remove this from config.

Reply:
I'll say secp384r1 then since it's the middle option.